recipes/procps/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+./autogen.sh
+./configure --disable-debug --disable-dependency-tracking --disable-silent-rules --prefix=${PREFIX}
```

Comment:
Normally like to do `"${PREFIX}"` for those building on their own machines in the event a space or other nasty character slips in.